tensorflow/python/estimator/training.py

```diff
@@ -771,13 +820,16 @@ def _start_continuous_evaluation(self):
     """Repeatedly calls `Estimator` evaluate and export until training ends."""
 
     _assert_eval_spec(self._eval_spec)
+    if self._eval_spec.throttle_secs is None:
```

Comment:
For the sake of consistency (i.e. being able to use same config for distributed training or local one) please also support the throttle_steps here. You can use Estimator.get_variable_value(...) to get latest step value.